pyseeyou/locales.py

```diff
@@ -1,23 +1,53 @@
 from __future__ import unicode_literals
 
+LOCALE_SEP = "-"
 
-def get_cardinal_category(num_string, lang):
-    n, i, v, w, f, t = get_parts_of_num(num_string)
 
-    if lang in LOCALE_FUNCTIONS:
-        return LOCALE_FUNCTIONS[lang](n, i, v, w, f, t)
+# Ported from https://github.com/format-message/format-message/blob/master/packages/lookup-closest-locale/index.js#L4-L15
+def lookup_closest_locale(locale, available):
```

Comment:
This function allows passing in both a list of locales, and a single string: a docstring would be great here to allow end users to easily see how this function works.